<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ASUS-Laptop" ignored="0" name="Test" tests="9" failures="1" timestamp="2020-11-02T20:17:35 CET" time="114.248" errors="0">
  <testcase name="log" time="3.108" classname="mycompany.login"/>
  <testcase name="log" time="4.997" classname="mycompany.login"/>
  <testcase name="signup" time="4.002" classname="mycompany.login"/>
  <testcase name="signup" time="4.022" classname="mycompany.login"/>
  <testcase name="signup" time="3.895" classname="mycompany.login"/>
  <testcase name="signup" time="3.996" classname="mycompany.login"/>
  <testcase name="signup" time="9.382" classname="mycompany.login">
    <failure type="java.lang.AssertionError" message="did not expect [Account successfully created.] but found [Account successfully created.]">
      <![CDATA[java.lang.AssertionError: did not expect [Account successfully created.] but found [Account successfully created.]
at mycompany.login.signup(login.java:59)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- signup -->
  <testcase name="signup" time="13.29" classname="mycompany.login"/>
  <testcase name="signup" time="6.883" classname="mycompany.login"/>
</testsuite> <!-- Test -->
